shopinvader_payment/services/invader_payment_service.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.addons.component.core import Component
+
+
+class InvaderPaymentService(Component):
+
+    _inherit = "invader.payment.service"
+
+    def _invader_find_payable_from_target(self, target, **params):
+        if target == "current_cart":
+            return self.component(usage="cart")._get()
+        return super()._invader_find_payable_from_target(target, **params)
```

Comment:
Maybe we should raise an error here in case that the cart do not exist. To avoid weird issue latter?
@sbidoul @lmignon

Reply:
Raising an error sounds good to me. UserError?